.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Be careful when you commit all the modified files in your folder, in this case you committed stuff of your Visual Studio Code env :)